### DISCUSSION TITLE
Feature Request: browser (chrome/edge) Add-on

### DISCUSSION THREAD

**Discussion by Snubletunge** [schlagmichdoch/PairDrop#343] [Ideas]
It would be highly useful to have Pairdrop available as an add-on in browsers, allowing quick and easy access at any time without needing to manually start the service. With an Edge add-on, users could:

	1.	Access Pairdrop directly from the toolbar, making file transfers faster and more efficient.
	2. Optionally set Pairdrop to run in the background or start automatically with browser for a seamless experience.
	3. Benefit from a streamlined setup for transfers between paired devices, whether on the same network or remotely.
	
Currently, Pairdrop can be used as a Progressive Web App (PWA) pinned to the toolbar, but this doesn’t offer the same functionality as a dedicated add-on. A true browser add-on would enhance usability and encourage more users to make Pairdrop a go-to tool for file transfers.

**Comment by g2rant** [comment]
I was just thinking the same thing. My main use for a browser extension would be to have an AirDrop option in the context menu when right-clicking on selected text or a link, which would allow me to quickly share that text to previously paired devices. 

Honestly I'm tempted to take this on myself. I've been wanting to try building extensions anyway for a couple other ideas.

**Comment by erikraft** [comment]
Just create a browser extension with **Iframe from pairdrop.net**


I know of a Browser Extension, but it is not from the PairDrop Website, but this Browser Extension is for File Transfer: [Firefox Browser Add-ons](https://addons.mozilla.org/pt-BR/firefox/addon/erikraft-drop/)

and there is one for VS Code that is also for File Transfer: [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=ErikrafT.erikraft-drop)

**PairDrop could do this**
